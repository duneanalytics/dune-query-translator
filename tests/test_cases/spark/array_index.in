select array_col[f(x) - 1]
