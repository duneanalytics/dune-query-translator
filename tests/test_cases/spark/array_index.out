/* Success! If you're still running into issues, check out https://dune.com/docs/query/syntax-differences/ or reach out in the #dune-sql Discord channel. */

/* !Array warning: Arrays in dune SQL are indexed from 1, not 0. The migrator will not catch this if you indexed using variables */

SELECT
  array_col[F(x) - 1]